pkg/pillar/types/locationconsts.go

```diff
@@ -132,6 +132,13 @@ const (
 	MemoryMonitorOutputDir = MemoryMonitorDir + "/output"
 	// MemoryMonitorPSIStatsFile - file to store memory PSI (Pressure Stall Information) statistics
 	MemoryMonitorPSIStatsFile = MemoryMonitorOutputDir + "/psi.txt"
+
+	// OVMFSettingsDir - directory for OVMF settings, they are stored in per-domain files
+	OVMFSettingsDir = PersistDir + "/ovmf"
```

Comment:
Do we care about a physically local attacker (who takes out disk, modifies it, and re-inserts it) being about to 1) read these settings and/or 2) modify them?

If so we should put this in /persist/vault/

Reply:
Sounds like a simple change that worth it.
And can we easily copy from this dir in runtime in this case? What's the technical difference between /persist/ and /persist/vault?

Reply:
Will it be a problem that the files are changed frequently by the running VM? They are not RO, and the guest OS really likes writing different EFI values in runtime. I see the hashes are changed from time to time.

Reply:
Will it be a problem that the files are changed frequently by the running VM? They are not RO, and the guest OS really likes writing different EFI values in runtime. I see the hashes are changed from time to time.

Reply:
Okay, @shjala explained to me that the directory is not under attestation. It's just an encrypted part of our FS, so when unlocked, it is accessible. So, if we assume the vault is unsealed at the moment we create the domain and shut it down, we should be fine.

Reply:
Fixed